nat-lab/tests/utils/vm/windows_vm_util.py

```diff
@@ -52,16 +59,43 @@ async def _copy_binaries(
         return
 
     try:
-        await connection.create_process(["rmdir", "/s", "/q", VM_TCLI_DIR]).execute()
-    except ProcessExecError as exception:
-        if (
-            exception.stderr.find("The system cannot find the file specified") < 0
-            and exception.stderr.find("The system cannot find the path specified") < 0
-        ):
-            raise exception
+        await connection.create_process(
+            ["taskkill", "/IM", "python.exe", "/F"]
```

Comment:
Killing python here, means it wasn't properly closed in previous session?

Reply:
Yes, the python process isn't killed when we stop telio, it's killed either after 180 seconds (manually configured to be longer than any test we might run without being unbounded). On linux and macos it's fine but windows complains. Running it detached like mentioned in a different comment and killing the process at the end of each tests like done for macos would be a more robust solution. I will try it out and see how it goes. Hopefully I can remove this